src/redux/reducers.js

```diff
@@ -423,6 +423,16 @@ export function commentViewState(state={}, action) {
       const viewStateMap = _.indexBy(commentsViewState, 'id')
       return {...viewStateMap, ...state}
     }
+    case response(SHOW_MORE_COMMENTS): {
+      const comments = action.payload.comments || []
+      const commentsViewState = comments.map(comment => ({
+        id: comment.id,
+        isEditing: false,
+        editText: comment.body
+      }))
+      const viewStateMap = _.indexBy(commentsViewState, 'id')
+      return {...viewStateMap, ...state}
+    }
```

Comment:
This case looks identical to the previous, "HOME". Could we maybe combine them together instead of copypasting? Unless there are arguments against it.